packages/types/index.d.ts

```diff
@@ -132,14 +137,18 @@ export interface UnitConfiguration {
   customViewUrl?: string;
 }
 
-export type SnapNetworks = 'polkadot' | 'kusama' | 'westend' | string;
+export type WellKnownSnapNetworks = 'polkadot' | 'kusama' | 'westend';
```

Comment:
Can we rename it as SupportedSnapNetworks as this will be a list of the networks that we verified internally

Reply:
Make sense, I have renamed it accordingly.